CHANGES.md

```diff
@@ -8,8 +8,14 @@
 * ancestral: Improvements to command line arguments. [#1344][] (@jameshadfield)
      * Incompatible arguments are now checked, especially related to VCF vs FASTA inputs. 
      * `--vcf-reference` and `--root-sequence` are now mutually exclusive.
+* translate: Tree nodes are checked against the node-data JSON input to ensure sequences are present. [#1348][] (@jameshadfield)
+* translate: The 'source' ID for GFF files is now ignored as a potential gene feature. [#1348][] (@jameshadfield)
+* translate: Improvements to command line arguments.  [#1348][] (@jameshadfield)
+    * `--tree` and `--ancestral-sequences` are now required arguments.
```

Comment:
I see this as a bug fix instead of feature. Without 704b3012f6377d5afcd353523e9b355dfef2a097, there would be unhandled exceptions.

Reply:
👍 [done](https://github.com/nextstrain/augur/compare/84343d81f7fff1929e7464401493aba8f0aacd3a..03c19657c12383cfffc81a7e933d269ade99ab99)